streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -1196,23 +1102,23 @@ public String toString() {
      * @return A string representation of the StreamThread instance.
      */
     public String toString(final String indent) {
-        final StringBuilder sb = new StringBuilder(indent + "StreamsThread appId: " + this.applicationId + "\n");
+        final StringBuilder sb = new StringBuilder(indent + "StreamsThread appId: " + applicationId + "\n");
```

Comment:
nit: I know this was here before, but why use a `StringBuilder` and then use `+` ?